adyenofficial.php

```diff
@@ -381,6 +392,43 @@ public function createWaitingForPaymentOrderStatus()
         return true;
     }
 
+    /**
+     * Create a new order status: "waiting for payment"
```

Comment:
Please update the comment based on the function

Reply:
updated!